api-specificatie/drc/openapi.yaml

```diff
@@ -1,20 +1,57 @@
 openapi: 3.0.0
 info:
   title: Documentregistratiecomponent (drc) API
-  description: Een API om een documentregistratiecomponent te benaderen
+  description: 'Een API om een documentregistratiecomponent te benaderen.
+
+
+    Het `INFORMATIEOBJECT` is het kernobject in deze API. Het is (digitaal)
```

Comment:
Een informatieobject is een digitaal document voorzien van meta-gegevens. Informatieobjecten kunnen aan andere objecten zoals zaken en besluiten worden gerelateerd (maar dat hoeft niet) en kunnen gebruiksrechten hebben. Gebruiksrechten leggen voorwaarden op aan het gebruik van het informatieobject (buiten raadpleging). Deze gebruiksrechten worden niet door API gevalideerd op gehandhaafd.

De API ondersteunt alleen enkelvoudige informatieobjecten.